test/helpers/customError.js

```diff
@@ -13,46 +10,41 @@ async function expectRevertCustomError(promise, expectedErrorName, args) {
       expect.fail('Expected 3rd array parameter for error arguments');
     }
     if (expectedErrorName) {
```

Comment:
Do we ever use this helper without an expected error name? Can we remove this condition?

Reply:
No, we never use it like that. I'm removing but let me double check for older errors